docs/dev/table/tableApi.md

```diff
@@ -627,8 +627,8 @@ Table result = left.join(right)
       </td>
     </tr>
     <tr>
-    	<td>
-        <strong>TableFunction Inner Join</strong><br>
+      <td>
+        <strong>Inner Join with User Defined Table Functions (UDTF)</strong><br>
```

Comment:
`User Defined` -> `User-Defined`

Reply:
Should this be actually called `User-Defined`? It can also be an temporal table function or some other built-in function, right? I think we should just use the term `TableFunction` as it was before.

Reply:
True, we can also drop user-defined.

Reply:
I was just matching it to be consistent with the phrasing used in `sql.md`. I can match it other way around as well.

Reply:
Hmmm, actually we do not have built in TFs and Temporal Joins will have their own section in the syntax so I will leave it as it is. Also this would require larger rework (separate independent issue to what I'm fixing), since all of the text in those paragraphs refers to UDTF's and how to register them.

Reply:
We have built-in table functions. `UNNEST` for example.